src/server/pachyderm_test.go

```diff
@@ -7403,6 +7404,75 @@ func TestListJobTruncated(t *testing.T) {
 	}, backoff.NewTestingBackOff()))
 }
 
+func TestListJobFilter(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+	c := tu.GetPachClient(t)
+
```

Comment:
Per slack, maybe add `DeleteAll()` here

Reply:
added it to the other ListJob* tests, too, for good measure